packages/bot-web-ui/src/components/dashboard/tutorial-tab/faq-content/faq-content.tsx

```diff
@@ -63,6 +60,13 @@ const FAQContent = observer(({ faq_list, hide_header = false }: TFAQContent) =>
         }, 5);
     };
 
+    const handleKeyboardEvent = e => {
+        const enterKeyCodes = ['Enter', 'Return', 13];
+        if (enterKeyCodes.includes(e.key) || enterKeyCodes.includes(e.keyCode)) {
```

Comment:
event.key === 'Enter' should be enough here. Can you please check this?

Reply:
done ✅